src/main/java/perficient/academic/universityapplication/controllers/UserController.java

```diff
@@ -0,0 +1,60 @@
+package perficient.academic.universityapplication.controllers;
+
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import perficient.academic.universityapplication.dtos.UserDto;
+import perficient.academic.universityapplication.mappers.UserMapper;
+import perficient.academic.universityapplication.models.User;
+import perficient.academic.universityapplication.services.UserService;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/users")
+@AllArgsConstructor
+@Getter
```

Comment:
Why you need a @Getter here?

Reply:
I'm using instead of userMapper getUserMapper, that is the reason to use @Getter, let me know if its better to use directly the property